CODEOWNERS

```diff
@@ -926,6 +926,7 @@ build.json @home-assistant/supervisor
 /homeassistant/components/radiotherm/ @bdraco @vinnyfuria
 /tests/components/radiotherm/ @bdraco @vinnyfuria
 /homeassistant/components/rainbird/ @konikvranik
+/tests/components/rainbird/ @konikvranik
```

Comment:
You should probably add yourself to codeowners given the scope of these changes

Reply:
Sending https://github.com/home-assistant/core/pull/84421

Reply:
Merged.